scenario/scenario_instance.json

```diff
@@ -0,0 +1,275 @@
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "georeference": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/"
+  },
+  "@id": "did:example:1234",
+  "@type": "scenario:scenario",
+  "scenario:common": {
+    "@type": "scenario:Common",
+    "scenario:abstractionLevel": "Logical",
+    "scenario:timeDate": {
+      "@value": "2024-05-27T10:00:00",
+      "@type": "xsd:dateTime"
+    },
+    "scenario:usedStandardFunctions": {
+      "@value": "TeleportAction,  SpeedAction, FollowTrajectoryAction",
+      "@type": "xsd:string"
+    },
+    "scenario:aim": {
+      "@value": "Test sensor perception range.",
+      "@type": "xsd:string"
+    }
+  },
+  "scenario:trafficParticipants": {
+    "@type": "scenario:TrafficParticipants",
+    "scenario:numberTrafficObjects": {
+      "@value": "1",
+      "@type": "xsd:unsignedInt"
+    },
+    "scenario:temporaryTrafficObjects": {
+      "@value": "0",
+      "@type": "xsd:unsignedInt"
+    },
+    "scenario:permanentTrafficObjects": {
+      "@value": "1",
+      "@type": "xsd:unsignedInt"
+    },
+    "scenario:movementDescription": {
+      "@value": "FollowTrajectory",
+      "@type": "xsd:string"
+    },
+    "scenario:controllers": {
+      "@value": "SampleControler1",
+      "@type": "xsd:string"
+    },
+    "scenario:customCommands": {
+      "@value": "UserDefinedAction: TurnAround",
+      "@type": "xsd:string"
+    }
+  },
+  "scenario:environmental": {
+    "@type": "scenario:Environmental",
+    "scenario:sunAzimuth": {
+      "@value": "3.1415",
+      "@type": "xsd:float"
+    }
+  },
+  "scenario:traffic": {
+    "@type": "scenario:Traffic",
+    "scenario:country": {
+      "@value": "Germany",
+      "@type": "xsd:string"
+    },
+    "scenario:countrySpecificTP": {
+      "@value": "CustomCar",
+      "@type": "xsd:string"
+    },
+    "scenario:countrySpecificSign": {
+      "@value": "Aircraft",
+      "@type": "xsd:string"
+    }
+  },
+  "scenario:structural": {
+    "@type": "scenario:Structural",
+    "scenario:catalogs": {
+      "@type": "general:Link",
+      "scenario:url": {
+        "@value": "https://inavlid.link.com/my_pedestrian_catalog.xosc",
+        "@type": "xsd:anyURI"
+      },
+      "scenario:type": "Document"
+    },
+    "": {
+      "@type": "general:Link",
+      "scenario:url": {
+        "@value": "https://inavlid.link.com/my_car_model.ogb",
+        "@type": "xsd:anyURI"
+      },
+      "scenario:type": "Document"
+    },
+    "scenario:trafficSpace": {
+      "@type": "general:Link",
+      "scenario:url": {
+        "@value": "https://inavlid.link.com/my_map.xodr",
+        "@type": "xsd:anyURI"
+      },
+      "scenario:type": "Document"
+    }
+  },
+  "scenario:format": {
+    "@type": "scenario:Format",
+    "scenario:dataFormat": "ASAM OpenSCENARIO",
+    "scenario:version": {
+      "@value": "1.2",
+      "@type": "xsd:string"
+    }
+  },
+  "general:general": {
```

Comment:
scenario:general and not general:general

Reply:
@heuerfin can you change this?

Reply:
done